python/tvm/relay/op/strategy/rocm.py

```diff
@@ -219,3 +221,93 @@ def batch_matmul_strategy_rocm(attrs, inputs, out_type, target):
             plevel=12,
         )
     return strategy
+
+
+def can_use_thrust(target, func_name):
+    return (
+        target.kind.name == "rocm"
+        and "thrust" in target.libs
+        and get_global_func(func_name, allow_missing=True)
+    )
```

Comment:
Thoughts on perhaps moving this helper to `python/tvm/contrib/rocm.py`? There are a few examples of helpers in python/tvm/contrib that help with op strategy dispatch based on conditions like these.

Reply:
Or given a few of your TODOs in this PR, like
```
# TODO(masahi): Check -libs=thrust option
```
maybe we want a common helper for use with both cuda and rocm?

Reply:
Yes, currently thrust dispatch is done in a ad hoc way, unlike cudnn/cublas etc where `-libs` option is required in addition to `USE_CUDNN=ON` etc. 

@tkonolige and I discussed this issue a bit and we agreed that we should enforce `-libs=thrust` to enable thrust offload too. For rocm, this requirement is already enforced in this PR, but for cuda it is not. That's what `TODO(masahi)` will going to fix in the next PR.

We can discuss what the common helper should be / where it should be in the next PR, would you agree?

Reply:
Works for me. :shipit: